client/src/components/_common/SectionHeader/SectionHeader.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './SectionHeader.module.css';
+
+/**
+ * A header for a `Section[…]` component
+ *
+ * - heading text
+ * - actions (e.g. links, buttons, form)
+ * - automatic styles or markup for given context (ex: within a form or a table)
+ *
+ * @example
+ * // a section header with heading text
+ * <SectionHeader>
+ *   <a href="…">Hyperlinked Name of Section</a>
+ * </SectionHeader>
+ * @example
+ * // a form header with actions and heading text
+ * <SectionHeader
+ *   actions={<button type="reset">Reset</button>}
+ *   isForForm
+ * >
+ *   Name of Form
+ * </SectionHeader>
+ * @example
+ * // a table header with actions and heading text
+ * <SectionHeader
+ *   actions={<input type="search" />}
+ *   isForTable
+ * >
+ *   Name of Table
+ * </SectionHeader>
+ */
+function SectionHeader({
+  actions,
+  children,
+  className,
+  isForForm,
+  isForTable
+}) {
+  let styleName = '';
+  const styleNameList = ['root'];
+  const HeadingTagName = isForForm || isForTable ? 'h3' : 'h2';
```

Comment:
There are sections that use different heading tags. I found the difference to be that forms use one tag, and tables use another. This is not demoed in UI Patterns. Ideally, section headers are consistent.